### DISCUSSION TITLE
Webdriverio 7 WebdriverIO.ElementArray is not recognised

### DISCUSSION THREAD

**Discussion by rossmasday** [webdriverio/webdriverio#6502] [Q&A] [answered]
I created a new v7 project and cannot find the import for ElementArray, in v6 the below code works but v7 the following error is shown on build:

_test/pageobjects/login.page.ts:14:34 - error TS2694: Namespace 'WebdriverIO' has no exported member 'ElementArray'.

get users(): WebdriverIO.ElementArray { return $$('#user'); }_

TSCONFIG:
_{
  "compilerOptions": {
    "outDir": "./.tsbuild/",
    "sourceMap": false,
    "target": "es2019",
    "module": "commonjs",
    "removeComments": true,
    "noImplicitAny": true,
    "strictPropertyInitialization": true,
    "strictNullChecks": true,
    "types": [
      "node",
      "webdriverio/sync",
      "@wdio/mocha-framework",
    ],
  },
  "include": [
    "./test/**/*.ts",
  ]
}_
Any help would be much appreciated.

**Comment by christian-bromann** [comment, marked as answer]
@rossimo thanks for raising this question. I addressed this in webdriverio/webdriverio#6521

**Reply by rossmasday** [reply]
@christian-bromann Brilliant thanks for your help!
